docker-compose.yml

```diff
@@ -8,27 +8,29 @@ services:
       - "MYSQL_ROOT_PASSWORD=password" 
       - "MYSQL_DATABASE=classes"
     volumes:
-      - sdschedule-data:/var/lib/mysql
+      - "sdschedule-data:/var/lib/mysql"
   sdschedule-backend:
     container_name: sdschedule-backend
     build: backend 
     ports:
       - "5000:5000"
-    volumes:
-      - "./backend:/app/"
```

Comment:
Correct me if I'm wrong.

From what I understand, in development mode, if any files change, our application will detect the changes and reload them.

For production, since we are [copying](https://github.com/CTrando/UCSD-Planner-Helper/blob/2c4bbeae802064527165c6eceae525cc3ce14183/backend/Dockerfile#L18)  those files into the container when building, I don't see the necessity to mount the real folder.

Reply:
Ah good, that's correct, thank you!